iotedgedev/cli.py

```diff
@@ -248,6 +248,42 @@ def deploy(manifest_file):
 main.add_command(deploy)
 
 
+@solution.command(
+    context_settings=CONTEXT_SETTINGS,
+    help="Create a deployment from the solution in IoT Hub")
+@click.option("--file",
+              "-f",
+              "manifest_file",
+              default=envvars.DEPLOYMENT_CONFIG_FILE_PATH,
+              show_default=True,
+              required=False,
+              help="Specify the deployment manifest file")
+@click.option("--name",
+              "-n",
+              required=True,
+              help="Specify the deployment name")
+@click.option("--priority",
+              "-p",
+              required=True,
+              help="Specify the deployment priority")
+@click.option("--target-condition",
+              "--tc",
+              "-t",
+              "target_condition",
+              default=envvars.get_envvar("LAYERED_DEPLOYMENT_TARGET_CONDITION"),
+              show_default=True,
+              required=False,
+              help="Specify the deployment target condition")
+@with_telemetry
+def deployment(manifest_file, name, priority, target_condition):
```

Comment:
@marianan I've renamed this command to `deployment` after our discussion yesterday about the confusion around layered vs complete (which both can be done using this command)

I did also consider `deployment create` or `deployment-create`, but thought `deployment` is not too unclear if you read the help flag.

What do you think?

PS: That reminds of a case we missed > complete deployment

Reply:
that looks good to me

Reply:
After discussing in a meeting we decided to go with: `iotedgedev iothub deploy`